Source/CarthageKit/Cartfile.swift

```diff
@@ -270,14 +270,18 @@ public struct Dependency<V: VersionType>: Equatable {
 	/// The version(s) that are required to satisfy this dependency.
 	public var version: V
 
-	public init(project: ProjectIdentifier, version: V) {
+	/// The dependencies of this dependency.
+	public let dependencies: Set<ProjectIdentifier>
```

Comment:
This is intended to be used for https://github.com/Carthage/Carthage/pull/972#issuecomment-162215783.